pkg/probe/ha-checksum.go

```diff
@@ -0,0 +1,42 @@
+package probe
```

Comment:
File should be named system_ha_checksum.go to match the API it uses.

Reply:
Perfect!